packages/platforms/accton/x86-64/x86-64-accton-as5916-54xm/onlp/builds/src/module/auto/x86_64_accton_as5916_54xm.yml

```diff
@@ -0,0 +1,50 @@
+###############################################################################
+#
+# x86_64_accton_as5916_54xm Autogeneration Definitions.
+#
+###############################################################################
+
+cdefs: &cdefs
+- x86_64_accton_as5916_54xm_CONFIG_INCLUDE_LOGGING:
```

Comment:
All of these config defines must be uppercase. 
Please s/x86_64_accton_as5916_54xm/X86_64_ACCTON_AS5916_54XM/g